KEP-5040: Disable git_repo volume driver. (#49946)
kubernetes/website: content/en/docs/concepts/storage/volumes.md

### gitRepo (deprecated) {#gitrepo}

{{< warning >}}

> The `gitRepo` volume plugin has been disabled by default.

Reviewer: ```suggestion
The `gitRepo` volume plugin is deprecated and is disabled by default.
```

Author: Done. Thanks!

> The plugin can be enabled by setting feature-gate GitRepoVolumeDriver=true.

Reviewer: ```suggestion
You can use this deprecated storage plugin in your cluster if you explicitly enable the `GitRepoVolumeDriver`
[feature gate](/docs/reference/command-line-tools-reference/feature-gates/).
```

I would move this sentence to be the first line after the warnings. We want people to read the warnings.

Author: Done. Thanks!
